src/any_agent/tools/mcp/frameworks/agno.py

```diff
@@ -16,30 +17,45 @@
     mcp_available = True
 
 
-class AgnoMCPServer(MCPServerBase):
-    """Implementation of MCP tools manager for Agno agents."""
+class AgnoMCPServerBase(MCPServerBase, ABC):
+    server: AgnoMCPTools | None = None
+    exit_stack: AsyncExitStack = AsyncExitStack()
```

Comment:
I'm a little worried about this change, since now all instantiations of AgnoMCPServer are going to share the same exitstack. Was this your intention?